owl/EASE-PROC.owl

```diff
@@ -234,12 +269,6 @@ In some sense any process that results in entities being created or destroyed mi
 
     <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-PROC.owl#Creation">
         <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-PROC.owl#ProcessType"/>
-        <rdfs:subClassOf>
```

Comment:
no they were invalid actually. Because they were rather supposed to link event and participant but we are here axiomatizing a *ProcessType* but not a *Process*.

Reply:
I assume someone has overseen this during refactoring before.